.gitignore

```diff
@@ -127,3 +127,5 @@ dmypy.json
 
 # Pyre type checker
 .pyre/
+
+practice_codes
```

Comment:
Please don't update the gitignore file.
You may use practice_codes as ignorable but others wont.